Extension/src/LanguageServer/extension.ts

```diff
@@ -30,6 +30,7 @@ nls.config({ messageFormat: nls.MessageFormat.bundle, bundleFormat: nls.BundleFo
 const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 export const cppBuildTaskProvider: CppBuildTaskProvider = new CppBuildTaskProvider();
 export const CppSourceStr: string = "C/C++";
+export const CppSourceStrPrefix: string = "C/C++: ";
```

Comment:
Is this the prefix of the task or launch config name?  I don't think "prefix" differentiates this much from the other string that doesn't have `: `.  I would call it `ConfigPrefix` or something like that.  (`Str` shouldn't really be in the name because we already know it's a string)